### PR TITLE
[CAT-2588] reimplement the tests in espresso

### PR DESCRIPTION
**Pull request by mpokrievka**
testInsertItemIntoUserListBrickMultipleLists() will fail until CAT-2359
is fixed.

### CODE REVIEW

**Comment by 84n4n4** [comment]
test this please

**Comment by 84n4n4** [comment]
please run espresso tests

**Comment by 84n4n4** [comment]
please change the commit message to something meaningful, theres a LOT of testing stories right now, please have message contain which test has been reimplemented

File: catroid/src/androidTest/java/org/catrobat/catroid/uiespresso/content/brick/app/InsertItemIntoUserListBrickTest.java [hunk 1]
```diff
@@ -0,0 +1,156 @@
...
+	private InsertItemIntoUserListBrick insertItemIntoUserListBrick;
+
+	@Rule
+	public BaseActivityInstrumentationRule<ScriptActivity> baseActivityTestRule = new
+			BaseActivityInstrumentationRule<>(ScriptActivity.class, true, false);
+
+	@Before
+	public void setUp() throws Exception {
+		brickPosition = 1;
+		insertItemIntoUserListBrick = new InsertItemIntoUserListBrick(10, 1);
```
Comment (84n4n4) [inline catroid/src/androidTest/java/org/catrobat/catroid/uiespresso/content/brick/app/InsertItemIntoUserListBrickTest.java]: please add variables for 10 and 1 and give them meaningful names.

File: catroid/src/androidTest/java/org/catrobat/catroid/uiespresso/content/brick/app/InsertItemIntoUserListBrickTest.java [hunk 2]
```diff
@@ -0,0 +1,156 @@
...
+		BrickTestUtils.createProjectAndGetStartScript("insertItemOfUserListBrickTest1")
+				.addBrick(insertItemIntoUserListBrick);
+		baseActivityTestRule.launchActivity(null);
+	}
+
+	@Category({Cat.AppUi.class, Level.Smoke.class})
+	@Test
+	@Flaky
```
Comment (84n4n4) [inline catroid/src/androidTest/java/org/catrobat/catroid/uiespresso/content/brick/app/InsertItemIntoUserListBrickTest.java]: is the flaky flag necessary for this test?

**Comment by 84n4n4** [comment]
please run espresso tests

**Comment by 84n4n4** [comment]
please squash commits.

**Comment by 84n4n4** [comment]
test this please

**Comment by 84n4n4** [comment]
please give this a use/meaningful commit message

File: catroid/src/androidTest/java/org/catrobat/catroid/uiespresso/content/brick/app/InsertItemIntoUserListBrickTest.java [hunk 3]
```diff
@@ -0,0 +1,163 @@
...
+import static junit.framework.Assert.assertEquals;
+
+import static org.catrobat.catroid.uiespresso.content.brick.utils.BrickDataInteractionWrapper.onBrickAtPosition;
+import static org.hamcrest.Matchers.allOf;
+import static org.junit.Assert.assertNotNull;
+
+@RunWith(AndroidJUnit4.class)
+public class InsertItemIntoUserListBrickTest {
+
+	private static int brickPosition;
```
Comment (84n4n4) [inline catroid/src/androidTest/java/org/catrobat/catroid/uiespresso/content/brick/app/InsertItemIntoUserListBrickTest.java]: why are the fields static?

**Comment by 84n4n4** [comment]
test this please

**Comment by 84n4n4** [comment]
please run espresso tests

**Comment by 84n4n4** [comment]
and please give this a proper commit message

**Comment by 84n4n4** [comment]
if the two new failing tests fail due to known bug, please add a comment at the point of failure with the ticket id of the bug.
if they should be succeeding, then something is going wrong with your test, please fix this.

**Comment by cheidenreich** [comment]
commit guidelines can be found here: https://confluence.catrob.at/display/KNOWHOW/Commit+Message+Guidelines

**Comment by cheidenreich** [comment]
closing this for now, please reopen once properly implemented

**Pull request closed without merging**